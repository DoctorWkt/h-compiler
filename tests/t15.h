void main()
{
  int x;

  for (x=00; x != 020; x++)
    printn(x);
  for (x=-10; x != -3; x++)
    printn(x);
}
