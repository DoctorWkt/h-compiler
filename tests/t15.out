000000
000001
000002
000003
000004
000005
000006
000007
000010
000011
000012
000013
000014
000015
000016
000017
777766
777767
777770
777771
777772
777773
777774
